server/server/src/config.rs

```diff
@@ -213,6 +214,36 @@ impl BillingConfig {
             ),
             google: GoogleConfig::from_env_vars(),
             stripe: StripeConfig::from_env_vars(),
+            apple: AppleConfig::from_env_vars(),
+        }
+    }
+}
+
+#[derive(Clone, Debug)]
+pub struct AppleConfig {
```

Comment:
Feels like we're out growing env vars, maybe we want to start using toml for config in the future.